src/rely/matchers/MockMatchers.re

```diff
@@ -1006,6 +1136,179 @@ module Make = (M: Mock.Sig) => {
           };
         });
 
+      let nthReturnedWith = isNot =>
+        createMatcher(
+          (
+            {matcherHint, formatReceived, formatExpected, indent},
+            actualThunk,
+            expectedThunk,
+          ) => {
+          let mock = actualThunk();
+          let (equals, n, expectedReturn) = expectedThunk();
+          if (n <= 0) {
+            (
+              () =>
+                "nth value "
+                ++ formatReceived(string_of_int(n))
+                ++ " must be greater than zero",
```

Comment:
So is this 1-indexed? Should it be 0 indexed instead? I would expect `nthReturnedWith(0)` to be the first one

Reply:
Yeah I made this 1-indexed. I'm pretty sure Jest is, and I think that this is how we speak about our expectations/is actually the user's expectation in this case.